### ISSUE TITLE
How to modify the default style

### ISSUE THREAD

**Issue by kkkasio** [jeanverster/chakra-ui-steps#27]
Hi! I really liked this addon!
I have a doubt about the customization of styles, following the chakra documents and as the library is created for the lib, I think I should follow the "theme edit" of the chakra itself...

Searching in some items of the author necessarily the following option.
`const Steps = {
  ...StepsStyleConfig,
  baseStyle: {
    // change styles here...
  },
};`
Example.
![image](https://user-images.githubusercontent.com/19254402/139127605-9512d886-b4c4-45b0-94e4-52f07767a0e2.png)

But on my front I get a restyle of the component's styling.
![image](https://user-images.githubusercontent.com/19254402/139128197-653845b7-58a3-487b-b142-67ac4e5fb37b.png)

I referred to modifying the theme in chackara, because modifying it in this way, it keeps the previous styles (default) of the component.
I'm confused if it's an error in my code or a bug.

**Comment by pianomansam** [comment]
This will soon be possible https://github.com/jeanverster/chakra-ui-steps/pull/23

**Comment by jeanverster** [comment]
Hey @kkkasio I have just released a new version `1.4.0` which fixes the error you're seeing above. Previously, any style customization would completely overwrite the existing styles, in the new version you should be able to customise any of the "parts" while still maintaining the original styles. For a list of the parts please see [here](https://github.com/jeanverster/chakra-ui-steps/blob/a11db0a42001ffb6ce39612880a2f6df168621bc/src/theme/index.ts#L9-L19) 

Using the above code should work as expected now :)

```js
const CustomSteps = {
  ...StepsStyleConfig,
  baseStyle: {
    steps: {
      color: 'red',
    },
    label: {
      color: 'red',
    },
  },
};

const theme = extendTheme({
  components: {
    Steps: CustomSteps,
  },
});
```
